_src/_who-we-are.jade

```diff
@@ -1,25 +1,57 @@
 div
-  h2.subheader Who we are
-  p
-  | Open Budget: Wichita is the result of many contributors including coders, community advocates, and city officials. We're looking for ideas and help, so get in touch and join in at our monthly
-  | meetups.  We have a monthly in-person meeting (please check 
-  a(href='http://www.meetup.com/openwichita/' target='_blank') meetup.com 
-  |for location and time)
-  | and on-line virtual meetings on Slack (
-  a(href='http://openwichita.com/slack' target='_blank') http://openwichita.com/slack
-  |)
-  h4.subheader Current team:
-  ul.list-unstyled
-    li Seth Etter, project lead
-    li ...
-    li (additional names)
-  h4.subheader City partners:
-  ul.list-unstyled
-    li ...
-  h4.subheader Contributors:
-  ul.list-unstyled
-    li the 
-      a(href='http://openwichita.com/', target='_blank') OpenWichita 
-      |  crew
-    li ...
-    li (additional names)
+h2.subheader Who we are
+p
+| Open Budget: Wichita is the result of many contributors including coders, community advocates, and city officials. We're looking for ideas and help, so get in touch and join in at our monthly
+| meetups.  We have a monthly in-person meeting (please check 
+a(href='http://www.meetup.com/openwichita/' target='_blank') meetup.com 
+|for location and time)
+| and on-line virtual meetings on Slack (
+a(href='http://openwichita.com/slack' target='_blank') http://openwichita.com/slack
+|)
+h4.subheader Current team:
```

Comment:
Let's replace this "Current Team" section with something that just links to the GitHub repo's contributors list or something. Maybe a quick paragraph about it originating in Oakland and then forked in OKC, then here. (Open Oakland, Code for OKC).